### DISCUSSION TITLE
Run unit tests in parallel but integration & e2e in band?

### DISCUSSION THREAD

**Discussion by alexandre-ducarne** [vitest-dev/vitest#4625] [Q&A]
Hi, Is it possible to select a set of tests to be run in parallel and others, which can interfere with each others, in band ?

For example, I have a mongodb instance for my whole test suite so one test can alter the db while another test was doing a find let's say.
To fix that, I would have to have one mongodb instance instantiated per test file and this would not be fast and take a good chunk of memory.

For now, I've split my tests with a suffix (.int, .e2e) and they each have their own vitest.config so I can decide if they run in threads or not. From this, I can run a different npm script with said configs for each type of test but I would prefer to simply call `npm test` and have everything working and reported in one shell

**Comment by sheremet-va** [comment]
No, it is not possible.
